### DISCUSSION TITLE
transcript links (and UI) removed in v3

### DISCUSSION THREAD

**Discussion by JuJuTheWulfPup** [discord-tickets/bot#327] [Support (Q&A)] [answered]
I was wondering why the transcript archiving and website/endpoint was removed in version 3. 
Personally I think it's really nice to dm users of a transcript of a closed ticket. Is there a reason it was removed?

Sorry if I should have been able to figure it out somehow within GitHub. I couldn't figure out where there may have been documentation that may list out the changes summarized (if such a place exists, please enlighten me). And even if it did find something, I didn't expect the reason to exist there.

**Comment by eartharoid** [comment, marked as answer]
In v3, text transcripts were moved to a (rubbish) [plugin](https://discordtickets.app/plugins/official/text-transcripts) - it sends a message when the ticket is closed but doesn't have the commands like v2.
The portal UI was supposed to be made and released soon after the bot but it never got done.

v4 will come with all of the features and commands that were in v2 but missing in v3, I'll start working on the new portal immediately after the bot is released.

**Reply by JuJuTheWulfPup** [reply]
Oh, that's great! I was just expecting a reason and not to be getting it back, and got the answer that I didn't even dare hope for.
Thanks for the quick response!
